src/engraving/libmscore/chordrest.cpp

```diff
@@ -400,6 +400,12 @@ EngravingItem* ChordRest::drop(EditData& data)
             bl->setGenerated(false);
 
             if (tick() == m->tick()) {
+                if (bl->barLineType() != BarLineType::START_REPEAT) {
+                    m = m->prevMeasure();
+                    if (!m) {
+                        return 0;
```

Comment:
I believe you would need to add `delete e;` above this line, like in other places in this method.
```suggestion
                        delete e;
                        return 0;
```

Reply:
Thanks, fixed now!